tests/helpers/runif.py

```diff
@@ -103,6 +105,7 @@ def __new__(
             tpu: Require that TPU is available.
             ipu: Require that IPU is available.
             hpu: Require that HPU is available.
+            mps: Require that MPS (M1 Metal GPU) is available or not available.
```

Comment:
For consistency. Also let's not mention M1 as there will likely be an M2 and beyond

```suggestion
            mps: Require that MPS (Apple Sillicon) is available.
```